README.md

```diff
@@ -14,73 +14,61 @@ make
 make check
 ```
 
-If that succeeds, run:
+If that succeeds, install with:
 
 ```
 sudo make install
 ```
 
-`--enable-pwdbased` is for password based encryption allowing the user
-to specify a unique password known only to him/her self and the
-recipient of the encrypted file.
++ `--enable-pwdbased` enables password based encryption.
++ `--enable-opensslextra` enables wolfSSL's OpenSSL compatibility layer.
++ `--enable-base64encode` enables Base64 encoding.
++ `--enable-keygen` enables key generation.
 
-`--enable-opensslextra` provides utility for a hex to binary conversion of
-hexidecimal values.
+Additional features that can be included when configuring wolfSSL for encryption or decryption are:
 
-`--enable-base64encode` enables Base64 encoding (not on by default)
++ --enable-camellia
++ --enable-des3
++ --enable-blake2
++ --enable-sha512
++ --enable-fortress
 
-`--enable-keygen` enables key generation (not on by default)
+## wolfCLU Installation
 
+After wolfSSL is installed, install wolfCLU.
 
-Additional features that can be included when configuring wolfssl for
-encryption or decryption are:
-
-        --enable-camellia
-        --enable-des3
-        --enable-blake2
-        --enable-sha512
-        --enable-fortress
-
-Additional features that can be included when configuring wolfssl for
-key generation are:
-
-
-
-##wolfCLU Install
-
-After wolfssl is installed, install wolfCLU.  In the directory
-`wolfssl-examples/wolfCLU` enter the following commands:
-
-    ./autogen.sh
-    ./configure
-    make
-    (optionally) make check OR make test
-    sudo make install
-
-Notes:
-* If `wolfssl` was recently installed run `sudo ldconfig` to update the linker cache.
+```
+./autogen.sh
```

Comment:
cd wolfCLU.... Also make a note that autogen.sh is only necessary if user got source from github.

Reply:
Will add a note that I'm assuming you're running this from the wolfCLU root.